aspnetcore/fundamentals/minimal-apis/samples/WebMinAPIs/Snippets/Program.cs

```diff
@@ -5,18 +5,18 @@ public static class Program
     public static void ManualRequestBinding(WebApplication app)
     {
         // <snippet_ManualRequestBinding>
-        app.MapGet("/{id}", (HttpContext context) =>
+        app.MapGet("/{id}", (HttpRequest request) =>
         {
-            var id = context.Request.RouteValues["id"];
-            var page = context.Request.Query["page"];
-            var customHeader = context.Request.Headers["X-CUSTOM-HEADER"];
```

Comment:
Do we have an example anywhere on how to read cookies?

Reply:
I can't find anything in the docset.

@Rick-Anderson I'd expect there to be something [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/app-state?view=aspnetcore-6.0#cookies), perhaps. Is there somewhere else that covers this?